lib/fog/sakuracloud/models/volume/disk.rb

```diff
@@ -38,6 +38,11 @@ def carve_hostname_on_disk(hostname)
           true
         end
 
+        def attach(server_id)
+          service.attach_disk(@attributes[:id], server_id )
```

Comment:
No need to use the instance variable `@attributes` since each attribute has it own getter.